Unix/installbuilder/GNUmakefile

```diff
@@ -133,25 +133,26 @@ ifneq ($(PF),Darwin)
 
 	mkdir -p $(PACKAGE_DIR)
 	cp $(OUTPUTDIR)/release/omi-* $(PACKAGE_DIR)
+endif # ifneq ($(PF),Darwin)
 
-else # ifneq ($(PF),Darwin)
-
+ifeq ($(PF),Darwin)
```

Comment:
I'm not sure what the difference is here, or was this just a cosmetic change for you?